framework/console/controllers/BaseMigrateController.php

```diff
@@ -41,6 +41,28 @@
      * or a file path.
      */
     public $templateFile;
+    /**
+     * @var array the template file for generating migration code automatically.
+     * This can be either a path alias (e.g. "@app/migrations/template.php")
+     * or a file path.
+     */
+    public $templateFileGenerators;
+    /**
+     * @var array the fields
+     */
+    public $fields;
+    /**
+     * @var string the primary key name
+     */
+    public $primaryKey = 'id';
+    /**
+     * @var string
+     */
+    public $createAt = 'create_at';
```

Comment:
@samdark  i remove primaryKey, createAt and updateAt attributes.

Now it only generate primary key if you dont specify primary key in fields option, for example : 

```
yii migrate/create CreatePost --fields=title:primaryKey
```
Generate : 
```php
$this->createTable('post', [
    'title' => $this->primaryKey()
]);
```

```
yii migrate/create CreatePost --fields=title:string
```
Generate : 
```php
$this->createTable('post', [
    'id' => $this->primaryKey(),
    'title' => $this->string()
]);
```